<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QWidget" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>169</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QPushButton" name="Confirm_pushButton">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Cancel_pushButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>132</y>
     <width>212</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="Tel_label">
      <property name="text">
       <string>电  话：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Tel_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>196</y>
     <width>159</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="Age_label">
      <property name="text">
       <string>年  龄：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="Age_comboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>68</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="Id_label">
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Id_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>164</y>
     <width>159</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="Agenda_label">
      <property name="text">
       <string>性  别：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="Agenda_comboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>36</y>
     <width>212</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="Name_label">
      <property name="text">
       <string>姓  名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Name_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>212</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="Password_label">
      <property name="text">
       <string>密  码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Password_lineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>228</y>
     <width>212</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="Address_label">
      <property name="text">
       <string>地  址：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Address_lineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Name_lineEdit</tabstop>
  <tabstop>Id_lineEdit</tabstop>
  <tabstop>Password_lineEdit</tabstop>
  <tabstop>Tel_lineEdit</tabstop>
  <tabstop>Agenda_comboBox</tabstop>
  <tabstop>Age_comboBox</tabstop>
  <tabstop>Address_lineEdit</tabstop>
  <tabstop>Confirm_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
